client/components/Assignments/DesksSubNavDropDown.jsx

```diff
@@ -28,21 +35,27 @@ export const DesksSubnavDropdown = ({userDesks, selectedDeskId, selectAssignment
         });
     }
 
-    items.push({divider: true});
+    if (showDeskAssignmentView) {
+        items.push({divider: true});
```

Comment:
This could be moved to the next if statement, no need to have two identical if statements